modules/evm/rpc/src/lib.rs

```diff
@@ -241,14 +257,15 @@ where
 			.map_err(|err| internal_err(format!("execution fatal: {:?}", err)))?;
 
 		// Determine the highest possible gas limits
-		let max_gas_limit = MAX_GAS_LIMIT;
-		let mut highest = U256::from(request.gas_limit.unwrap_or(max_gas_limit));
+		let max_gas_limit = request.max_gas_limit;
+		let max_storage_limit = request.max_storage_limit;
+		let mut highest = std::cmp::max(request.gas_limit, max_gas_limit);
```

Comment:
This will cause each transaction to be estimated from the maximum gas_limit, increasing the number of executions

Reply:
@zjb0807 this is supposed to be `cmp::min`, fixing it